components/Input/index.d.ts

```diff
@@ -41,4 +41,15 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   };
 }
 
-export default class Input extends Component<InputProps> {}
+interface CompoundedInput extends React.FunctionComponent<InputProps> {
```

Comment:
```suggestion
interface CompoundedInput extends React.Component<InputProps> {
```
Input is not a functional component